matrix/results/bun.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "bun",
+  "bin": "node",
+  "args": [
+    "/home/naugtur/repo/endo-e2e-tests/matrix/cases/import-cjs/dist/bun.mjs"
```

Comment:
Non-portable fully-qualified path.

Reply:
these are just documenting what happened during the run. But I turned them into relative paths to get rid of this.